src/CHANGES.txt

```diff
@@ -10,6 +10,9 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
 
     - Whatever John Doe did.
 
+  From Mathew Robinson:
+
+    - Fix issue #3415 - Update remaining usages of EnvironmentError to SConsEnvironmentError
```

Comment:
Can you add a blurb under which conditions it would fail? looks like cachedir not writeable, and some others. So folks would know 3.1.1 solves such?